app/controllers/evidence_controller.rb

```diff
@@ -5,6 +5,7 @@ class EvidenceController < NestedNodeResourceController
   include MultipleDestroy
   include NodesSidebar
   include NotificationsReader
+  include EvidenceHelper
```

Comment:
I have a feeling eventually we'll end up with our own routes helper file. We do enough switches to generate paths it'll be helpful. I'm good with this as is for now.